activesupport/lib/active_support/ordered_options.rb

```diff
@@ -84,6 +84,12 @@ def inspect
   #   h = ActiveSupport::InheritableOptions.new({ girl: 'Mary', boy: 'John' })
   #   h.girl # => 'Mary'
   #   h.boy  # => 'John'
+  #
+  # If the existing hash has string keys, use ActiveSupport::HashWithIndifferentAccess#with_indifferent_access on it.
+  #
+  #   h = ActiveSupport::InheritableOptions.new({ 'girl' => 'Mary', 'boy' => 'John' }.with_indifferent_access)
```

Comment:
Instead of `with_indifferent_access`, let's suggest `symbolize_keys`.  That emphasizes that the lookup is performed with symbols, and it is a bit more efficient than using `HashWithIndifferentAccess`.

```suggestion
  # If the existing hash has string keys, call Hash#symbolize_keys on it.
  #
  #   h = ActiveSupport::InheritableOptions.new({ 'girl' => 'Mary', 'boy' => 'John' }.symbolize_keys)
```

Reply:
Good call